docs/user-guide/data.md

```diff
@@ -219,44 +221,86 @@ themselves using the `lfs quota` command. To do this:
+The solid state storage file system is a 1 PB high performance parallel Lustre file system
+similar to the /work file systems. However, unlike the /work file systems, all of the
+disks are based solid state storage (NVMe) technology. This changes the performance characteristics of the 
+file system compared to the /work file systems. Testing by the ARCHER2 CSE team at
+EPCC has shown that you may see I/O performance improvements from the solid state
+storage compared to the standard Lustre file systems on ARCHER2 if your I/O model
+has the following characteristics or similar:
 
-The data that was available on the RDF `/epsrc` and `/general` file systems
-is available on ARCHER2 via the RDFaaS (RDF as a Service) file system. If 
-you have requested the same username in the same project as you had on 
-ARCHER then you will be able to access your data at either:
+- You read/write lots of files in parallel (e.g. your software uses a file-per-process
+  model or similar)
+- You use the [ADIOS 2](https://adios2.readthedocs.io/en/latest/) I/O system
+
+Data on the solid state (NVMe) file system is visible on the compute nodes
+
+!!! important
+    If you use MPI-IO approaches to reading/writing data - this includes parallel HDF5
+    and parallel NetCDF - then you very unlikely to see any performance improvements
+    from using the solid state storage over the standard parallel Lustre file systems
+    on ARCHER2.
+
+!!! warning
+    There are no backups of any data on the work file systems. You should
+    not rely on these file systems for long term storage.
+
+#### Access to the solid state file system
+
+Projects do not have access to the solid state file system by default. If you would like
+access to the solid state file system, you should contact the
+[ARCHER2 Service Desk](https://www.archer2.ac.uk/support-access/servicedesk.html) with a
+short description of why you want access, how much space you require and how long you
+require access for.
+
+!!! important
+    While we will endeavour to meet all reasonable requests for access, the capacity of the
+    solid state file system is limited and we may not be able to fulfil all requests.
+
+#### Location of directories
+
+If your project has been granted access to the solid state file system, you can find
+your directory on the file system at:
 
 ```
-/epsrc/<project-code>
+/mnt/lustre/a2fs-nvme/work/<project code>/<project code>/<username>
 ```
 
-or
+For example, if my username is `auser` and I am in project `t01`, I could find my 
+solid state storage directory at:
 
 ```
-/general/<project-code>
+/mnt/lustre/a2fs-nvme/work/t01/t01/auser
 ```
 
-depending on which file system it was in on the RDF file systems. For example,
-if your username is `auser` and you are in the `e05` project, then your RDF data
-will be on the RDFaaS file system at:
+#### Quotas on solid state file system
+
+You query quotas for the solid state file system in the same way as for the Work file
```

Comment:
May be worth having a link to the relevant quota section here. (And you may want `/work`.)

Reply:
I have added an internal link to the quota information for the work file systems. I have also tidied up the work terminology to make it consistent throughout the document (most places used "work" without the "/" so I went for this)